### PR TITLE
Adding Virtualhere server

### LINKED ISSUE #31562: Cask request: VirtualHere USB Server
### Cask details

Name: VirtualHere USB Server

Homepage: https://www.virtualhere.com/osx_server_software

Download URL: https://www.virtualhere.com/sites/default/files/usbserver/VirtualHereServer.dmg

Description: Remotely access your iLok, printer, disk, iPhone, USB-Serial device and most USB devices connected to your Mac remotely over the network, just as if the device was connected directly!

### PR DESCRIPTION
**Pull request by gobinathm**
- New Cask to Include VirtualHere USB Server

Closes #31562

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/virtualhere-server.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'virtualhere-server' do
+  version '3.2.8'
```
Comment (victorpopkov) [inline Casks/virtualhere-server.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`.

File: Casks/virtualhere-server.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'virtualhere-server' do
```
Comment (victorpopkov) [inline Casks/virtualhere-server.rb]: The [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md) should be followed.

In this case, the both filename and cask header should be named as `virtualhereserver`.

**Review by victorpopkov — CHANGES_REQUESTED** [review]

File: Casks/virtualhere-server.rb (hunk 2 above)
Comment (gobinathm) [inline Casks/virtualhere-server.rb]: @victorpopkov Its updated now

File: Casks/virtualhere-server.rb (hunk 1 above)
Comment (gobinathm) [inline Casks/virtualhere-server.rb]: Change this as well

**Comment by gobinathm** [comment]
@victorpopkov Update the cask

**Pull request merged**